webapp/src/components/AssetPage/SaleRentActionBox/SaleRentActionBox.container.ts

```diff
@@ -21,20 +21,18 @@ const mapState = (state: RootState, ownProps: OwnProps): MapStateProps => {
     userHasAlreadyBidsOnNft: wallet
       ? getNFTBids(state).some(bid => bid.bidder === wallet.address)
       : false,
-    isOwner: isOwnedBy(ownProps.nft, wallet)
+    isOwner: isOwnedBy(
+      ownProps.nft,
+      wallet,
+      ownProps.rental ? ownProps.rental : undefined
```

Comment:
nit pick: 
```suggestion
      ownProps.rental ?? undefined
```
so it's used as the [one below](https://github.com/decentraland/marketplace/pull/976/files#diff-ee8c189daceb7fc88075336992dfe533703e2db7f881963d684458252eddd6c3R90)